setup.py

```diff
@@ -35,7 +35,13 @@
         "airflow_tecton.sensors",
         "airflow_tecton.operators",
     ],
-    install_requires=["apache-airflow>=2.0", "requests"],
+    install_requires=["apache-airflow>=2.0",
+                      "requests",
+                      "pandas==1.3.5",
+                      "pydantic==1.10.9",
```

Comment:
do we actually need this?

Reply:
yes, airflow doesn't work w/ latest pydantic

Reply:
is pydantic actually required by airflow? regardless, it should be pinned by >=, not by `==` which will cause problems with customer deps

Reply:
it is either required by airflow or by tecton.  ">=" doesn't work, since pip auto picks up highest version, which fails airflow start.

Reply:
i think maybe you need to pin above and below, because it is not acceptable in python to pin an exact dep in setup.py

Reply:
honestly I would lean towards not pinning it. Airflow not working with latest pydantic is an airflow problem that they should fix, not us